src/current/_includes/releases/v24.1/v24.1.1.md

```diff
@@ -0,0 +1,103 @@
+
+- The  [optimizer]({% link {{ page.version.version }}/cost-based-optimizer.md %}) can now plan constrained scans over partial indexes in more cases, particularly on partial indexes with predicates referencing [virtual computed columns]({% link {{ page.version.version }}/computed-columns.md %}#virtual-computed-columns). [#123468][#123468]
+- The storage parameter `ttl_delete_rate_limit`, which determines the rate limit for deleting expired rows, is now set to `100` by default. [#124354][#124354]
+- CockroachDB no longer limits precision when converting [spatial data types]({% link {{ page.version.version }}/architecture/glossary#data-types.md %}) to JSON. [#124536][#124536]
+- The `optimizer_push_offset_into_index_join` [session setting]({% link {{ page.version.version }}/session-variables.md %}) has been added. When enabled, the  [optimizer]({% link {{ page.version.version }}/cost-based-optimizer.md %}) will attempt to push offset expressions into index join expressions to produce more efficient query plans. The setting is enabled by default on v24.1 and later, and disabled on v23.2. [#124490][#124490]
+- The default value of the `sql.defaults.results_buffer.size` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) has been changed from 16KiB to 512KiB. This reduces the chance that clients using [`READ COMMITTED`]({% link {{ page.version.version }}/read-committed.md %}) transactions will hit errors that cannot automatically be retried within CockroachDB. [#124633][#124633]
+
+<h3 id="v24-1-1-operational-changes">Operational changes</h3>
+
+- The default limit for in-memory statement and transaction fingerprints has been lowered from 100,000 to 5000. This affects the [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}) `sql.metrics.max_mem_stmt_fingerprints` and `sql.metrics.max_mem_txn_fingerprints`. [#123430][#123430]
+- Added the `sql.pgwire.pipeline.count` [metric]({% link {{ page.version.version }}/metrics.md %}), which is a gauge that measures how many wire protocol commands have been received by the server, but have not yet begun processing. This metric will only grow if clients are using the "pipeline mode" of the PostgreSQL wire protocol. [#124256][#124256]
+- The `client_authentication_ok` and `client_session_end` messages are now logged to the `SESSIONS` [log channel]({% link {{ page.version.version }}/logging-overview.md %}#logging-channels) unconditionally. Previously, these would only be logged if the `server.auth_log.sql_sessions.enabled` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) was set to `true`. All other `SESSIONS` log messages are still only logged if `server.auth_log.sql_sessions.enabled` or `server.auth_log.sql_connections.enabled` are set to `true`. If you do not want to see `client_authentication_ok` or `client_session_end` messages, then the `SESSIONS` log channel can be disabled entirely. [#124369][#124369]
```

Comment:
```suggestion
- The `client_authentication_ok` and `client_session_end` messages are now logged to the `SESSIONS` [log channel]({% link {{ page.version.version }}/logging-overview.md %}#logging-channels) unconditionally. Previously, these would be logged only if the `server.auth_log.sql_sessions.enabled` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) was set to `true`. All other `SESSIONS` log messages are logged only if `server.auth_log.sql_sessions.enabled` or `server.auth_log.sql_connections.enabled` are set to `true`. To prevent logging of `client_authentication_ok` or `client_session_end` messages, you can optionally disable the `SESSIONS` log channel. [#124369][#124369]
```